### ISSUE TITLE
Invalid form body

### ISSUE THREAD

**Issue by hacktek** [v0idp/Mellow#92]
Currently on Mellow v2.7.33.

When searching for "!movie the lord of the rings" the following error is generated:

```
DiscordAPIError: Invalid Form Body


embed.fields[0].value: Must be 1024 or fewer in length.


    at RequestHandler.execute (/usr/src/app/node_modules/discord.js/src/rest/RequestHandler.js:170:25)


    at processTicksAndRejections (internal/process/task_queues.js:94:5) {


  name: 'DiscordAPIError',


  message: 'Invalid Form Body\nembed.fields[0].value: Must be 1024 or fewer in length.',


  method: 'post',


  path: '/channels/499307826204770316/messages',


  code: 50035,


  httpStatus: 400


}
```

Searching for just "lord" does not have the same results and neither does searching for another long name such as "the curious case of benjamin button".

**Comment by v0idp** [comment]
Was able to reproduce this. Will look into it and fix it. thank you

**Comment by v0idp** [comment]
I was able to find the bug. will be fixing it soon.

**Comment by v0idp** [comment]
Fixed in fab9e88

**Comment by hacktek** [comment]
Confirmed fixed. 

Thanks!
